client/db/bolt/upgrades.go

```diff
@@ -97,8 +100,8 @@ func v1Upgrade(dbtx *bbolt.Tx) error {
 	const newVersion = 1
 
 	dbVersion, err := fetchDBVersion(dbtx)
-	if err == nil {
-		return fmt.Errorf("unexpected database version found")
+	if err != nil {
+		return fmt.Errorf("error fetching database version: %w", err)
```

Comment:
There would be no value for versionKey with the initial _implicitly_ v0 database, so this should return nil.

Reply:
This was supposed to be a draft comment while I checked this out.  In fact my entire review was not supposed to be submitted but github submitted it anyway when I tried to make [a regular comment outside of review](https://github.com/decred/dcrdex/pull/704#discussion_r497788085).

Reply:
Did you figure this out then? [We've been adding the current version](https://github.com/decred/dcrdex/blob/102a392ca4e05a8bbc53aa6260a133e36951136b/client/db/bolt/db.go#L120-L131) to newly created db files for some time now, even though we disabled the initial upgrade. So unless someone is trying to run with a DB file from about 3 months back, they are certain to have a version number at the `versionKey`.